modules/openapi-generator/src/main/resources/typescript/http/http.mustache

```diff
@@ -2,7 +2,7 @@
 {{#node}}
 // TODO: evaluate if we can easily get rid of this library
 import {{^supportsES6}}* as{{/supportsES6}} FormData from "form-data";
-import { URLSearchParams } from 'url';
+import { URL, URLSearchParams } from 'url';
 import * as http from 'http';
 import * as https from 'https';
 {{/node}}
```

Comment:
Can the code below be removed?
https://github.com/OpenAPITools/openapi-generator/blob/9fe05a4c87357bca3e1e45226d088bea0632e1b9/modules/openapi-generator/src/main/resources/typescript/http/http.mustache#L10-L14

and 

https://github.com/OpenAPITools/openapi-generator/blob/9fe05a4c87357bca3e1e45226d088bea0632e1b9/modules/openapi-generator/src/main/resources/typescript/http/http.mustache#L60-L96

Reply:
yep!